Vagrantfile

```
@@ -13,6 +13,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.define "xenial" do |xenial|
     xenial.vm.box = "ubuntu/xenial64"
   end
+  
+  config.vm.define "bionic" do |bionic|
+    xenial.vm.box = "ubuntu/bionic64"
```

Typo, that causes an error when running the `ckan-package` command
```suggestion
    bionic.vm.box = "ubuntu/bionic64"
```